### PR TITLE
Feature/hazel partner

### PR DESCRIPTION
**Pull request by Palmdate**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+
+def sortable(column, title = nil)
+  current_status = params[:status]
+  title ||= column.titleize
+  css_class = column == sort_column ? "current #{sort_direction}" : nil
+  direction = column == sort_column && sort_direction == "asc" ? "desc" : "asc"
+  link_to title, {:sort => column, :direction => direction, :status => current_status }, {:class => css_class}
+end
+
+def sortran(column, title = nil)
```
Comment (lhhieu89) [inline app/helpers/application_helper.rb]: What is `sortran` mean?

File: config/routes.rb [hunk 2]
```diff
@@ -23,6 +23,7 @@
 
   get 'partner/transaction_history' => 'partner#trans_history'
   get 'partner/faq' => 'faqs#index'
+  get 'partner/students/filter_by_status' => 'partner/students#filter_by_status'
```
Comment (lhhieu89) [inline config/routes.rb]: Please remove this line

File: app/views/partner/students/_filter.html.slim [hunk 3]
```diff
@@ -1,8 +1,26 @@
...
+  - else
+    = link_to "SERVING", partner_students_path(status: 'serving'), class: 'btn btn-filter'
+  - if current_status == 'completed'
+    = link_to "COMPLETED", partner_students_path(status: 'completed'), class: 'btn btn-filter btn-active'
+  - else
+    = link_to "COMPLETED", partner_students_path(status: 'completed'), class: 'btn btn-filter'
+  - if current_status == 'refund'
+    = link_to "REFUND", partner_students_path(status: 'refund'), class: 'btn btn-filter btn-active'
+  - else
+    = link_to "REFUND", partner_students_path(status: 'refund'), class: 'btn btn-filter'
```
Comment (lhhieu89) [inline app/views/partner/students/_filter.html.slim]: Can you use if inline?

File: config/routes.rb (hunk 2 above)
Comment (Palmdate) [inline config/routes.rb]: ok @lhhieu89

File: app/views/partner/students/_filter.html.slim (hunk 3 above)
Comment (Palmdate) [inline app/views/partner/students/_filter.html.slim]: I have commited code shorter than that

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (Palmdate) [inline app/helpers/application_helper.rb]: I must have new method of sort on history transaction beacause with sortable method I add status attribute and with history of transaction don't need

File: app/views/partner/students/_filter.html.slim (hunk 3 above)
Comment (lhhieu89) [inline app/views/partner/students/_filter.html.slim]: Which commit?

**Pull request merged**